Core kit 2.3.5 (#103)
bottlerocket-os/bottlerocket-core-kit: CHANGELOG.md

# v2.3.5 (2024-08-21)

## Orchestrator Changes

### Kubernetes

>  * kubernetes (all): handle null in credential provider template ([#101])

Reviewer: We can either list the versions explicitly or leave the implication it applies to all. 
```suggestion
 * Fix issue where a null value would fail to render the credential provider template for Kubernetes ([#101])
```

## OS Changes

>  * ghostdog: remove /dev/ from device name ([#98])
>  * kernels: update to 5.10.223-212 and 6.1.102-111 ([#99])
>  * udev: update symlink path for EBS mappings ([#98])

Reviewer: The first and third were in the same PR, let's talk about them together.
```suggestion
 * Improve EBS volume udev rules by adding a symlink to `/dev/by-ebs-id` and remove `/dev/` the device name returned by ghostdog ([#98])
 * Update kernels to 5.10.223-212 and 6.1.102-111 ([#99])
```
Or we can keep them as two lines, but the PR is the same, it makes this seem like they were done separately as is.

## tools

>  * tools: add collect-kernel-config script ([#84])

Reviewer: ```suggestion
 * Add collect-kernel-config script to tools ([#84])
```
